source/custom_templates/_tx_api_page.txt

```diff
@@ -1,8 +1,9 @@
 .. include:: _:page:.tmp
 
-.. include:: common_response_params.rst
+.. include:: _:org_type:_param_table.rst
 
-.. include:: optional_fields.txt
+The following fields are optional based on your account type:
 
+.. include:: _:org_type:_optional_param_table.rst
 
```

Comment:
Thanks for the clarification. 
So 1) Is not a problem but 2) is because one org_type needs to be catered to specially. 
I'll find a way!